src/utils/geometry-vs-texture-metrics.ts

```diff
@@ -0,0 +1,88 @@
+import type { Tile3D } from "@loaders.gl/tiles";
```

Comment:
I suggest moving this file in `attributes-validation` folder.

Reply:
done